todo_app/e2e_test.py

```diff
@@ -8,37 +8,56 @@
 from todo_app.app import create_app
 from dotenv import load_dotenv, find_dotenv
 import time
+import pymongo
 
 
 def create_trello_board():
-
-    key = os.environ['KEY']
-    token = os.environ['TOKEN']
-    board_response = requests.post(
-        f'https://api.trello.com/1/boards/?name=test&key={key}&token={token}')
-    board_id = board_response.json()["id"]
-    response = requests.get(
-        f'https://api.trello.com/1/boards/{board_id}/lists?key={key}&token={token}')
-
-    lists = response.json()
-
-    for list in lists:
-
-        if list['name'] == "TODO":
-            os.environ['TOID'] = list['id']
-        elif list['name'] == "DOING":
-            os.environ['DOINGID'] = list['id']
-        elif list['name'] == "DONE":
-            os.environ['DONE'] = list['id']
-    return board_id
+    dbconnect = os.environ['CLIENT']
+    client = pymongo.MongoClient(
+        dbconnect, ssl=True, ssl_cert_reqs='CERT_NONE')
+    print(client.list_database_names())
+    database = client["01"]
+    test_collection = database["test_collection"]
+    # board_response = requests.post(
+    #     f'https://api.trello.com/1/boards/?name=test&key={key}&token={token}')
+    # board_id = board_response.json()["id"]
+    # response = requests.get(
+    #     f'https://api.trello.com/1/boards/{board_id}/lists?key={key}&token={token}')
+
+    # lists = response.json()
+
+    # for list in lists:
+
+    #     if list['name'] == "TODO":
+    #         os.environ['TOID'] = list['id']
+    #     elif list['name'] == "DOING":
+    #         os.environ['DOINGID'] = list['id']
+    #     elif list['name'] == "DONE":
+    #         os.environ['DONE'] = list['id']
```

Comment:
Let's delete it rather than comment it out - we can always recover it with git!

Reply:
Updated,thank you